app/controllers/projects_controller.rb

```diff
@@ -66,10 +66,20 @@ def update
 
     if params[:project][:rerun] == 'reducers'
       rerun_reducers
-      respond_with project, location: project_path(@project, anchor: 'reducers')
+      respond_with project, location: project_path(project, anchor: 'reducers')
     else
+      was_paused = project.paused?
       project.update(project_params.except('display_name'))
-      respond_with project
+
+      if !was_paused && project.paused?
```

Comment:
How is this not just `if !project.paused?` ?

Reply:
because the pause/unpause is through the normal update endpoint, we don't want to tell them again that we're pausing the project if we're just toggling the public reductions on and off, and both code paths go through here. so we really only want to display the "pausing" message if the state changed from active to paused.